stdlib/2/wsgiref/__init__.pyi

```diff
@@ -0,0 +1,26 @@
+# Type declaration for a WSGI Function in Python 2
+#
+# This function actually exist, but we need a central place to define the type
```

Comment:
I think you mean something along the lines of "WSGIFunction doesn't exist in wsgiref/__init__.py, it's a type provided for type checking purposes."